packages/branding/README.md

```diff
@@ -1,11 +1,41 @@
-# Taiko branding
+# Taiko Brand Resources
 
-This package contains all you need for Taiko branding.
+This package contains all the branding assets for Taiko. 
+
+Included: Logos, Logotypes, Backgrounds, Backdrops and Mockups.
 
 ## Naming
 
-For naming, we use Taiko or Taiko Labs. For the handle, we refer to Taiko with `taikoxyz`, which is where you can find us (ie. twitter: @taikoxyz, github: @taikoxyz). Please maintain this convention.
+For naming, we use Taiko or Taiko Labs. For our social media handle, we refer to Taiko with `taikoxyz`, which is where you can find us (ie. twitter: [@taikoxyz](https://twitter.com/taikoxyz), github: [@taikoxyz](https://github.com/taikoxyz)). Please maintain this convention.
```

Comment:
```suggestion
Use `Taiko` or `Taiko Labs` for the name.

Use `taikoxyz` for the social handle (e.g., Twitter: [@taikoxyz](https://twitter.com/taikoxyz), GitHub: [@taikoxyz](https://github.com/taikoxyz)).
```
ref: (active voice) https://learn.microsoft.com/en-us/style-guide/global-communications/writing-tips
ref: (don't use we) https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we